CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.22)
+project(geographiclib_cmake_module)
```

```suggestion
project(geographiclib_cmake_module NONE)
```

Since we don't compile anything in this package, setting the type to NONE skips all of the compiler sensing steps during CMake configuration.